apps/golden-sample-app/src/app/auth/auth-events-handler/auth-events-handler.service.spec.ts

```diff
@@ -70,7 +70,7 @@ describe('AuthEventsHandlerService', () => {
         flush();
       });
 
-      expect(oAuthService.revokeTokenAndLogout).toHaveBeenCalledTimes(1);
+      expect(oAuthService.logOut).toHaveBeenCalledTimes(1);
```

Comment:
Can you update test names please (and the describe) to remove the info about revoking tokens?